README.md

```diff
@@ -49,22 +48,25 @@ Vita3K is licensed under the **GPLv2** license. This is largely dictated by exte
     * [vita3k-git](https://aur.archlinux.org/packages/vita3k-git)<sup><small>AUR</small></sup>
   * Requirements:
     * xdg-desktop-portal
+* [Android](https://github.com/Vita3K/Vita3K-Android/releases)
 * Others
   * [Download Artifact](https://github.com/Vita3K/Vita3K/actions?query=event%3Apush+is%3Asuccess+branch%3Amaster)
 
 ## Building
 
-Please see [`building.md`](./building.md).
+Please see [`Building.md`](./documentation/Building.md).
 
 ## Running
-Specify the path to a .vpk file as the first command line argument, or run `Vita3K --help` from the command-line for a full list of options.
-For more detailed instructions on running/installing games on all platforms, please read the **#info-faq** channel on our [Discord Server](https://discord.gg/MaWhJVH).
+
+Install your game in .pkg or .vpk format and click the play button on the game's menu.
+For more detailed instructions on running/installing games on all platforms, please read [`Running_Games.md`](./documentation/Running_Games.md).
```

Comment:
pkg (commercial) vpk (homebrew only)
To avoid questions why vitamin dump not works

Reply:
fixed